README.md

```diff
@@ -9,7 +9,7 @@ Camdram is an [open source](http://opensource.org/docs/osd) project developed by
 
 The steps required to set up a development checkout of Camdram are detailed below. For the sake of brevity, these instructions assume that the reader is familiar with a number of technologies, such as developing on a Linux based platform, using Git and GitHub.
 
-New releases are made on the `master` branch every so often using the GitHub interface. This automatically generates a changelog and pushes the new release into production at https://www.camdram.net/. The latest changes can be seen at https://development.camdram.net/ which always reflects the `HEAD` state of this repository.
+New releases are made on the `master` branch every so often using the GitHub interface. This automatically generates a changelog and pushes the new release into production at https://www.camdram.net/. The latest changes can be seen [here](https://www.camdram.net/development/) which always reflects the `HEAD` state of this repository.
```

Comment:
https://www.camdram.net/development/ is something else. Either `development.camdram.net` should be made to stay up reliably or we should remove the link entirely.